card/mod/standard/set/all/rich_html/error/debug_server_error.haml

```diff
@@ -0,0 +1,1015 @@
+
+:css
+  /* Basic reset */
```

Comment:
wondering if this should be in a separate mod?  kinda massive and nonstandard.

Reply:
A good idea, in theory, but not trivial to realize. There are related coffeescript changes in the slot stuff and changes for the ServerError exception. There is also the questions if the "shark" debugger is really useful and what to do with ajax request. I have to catch the exception if I want to show it nicely formatted in a modal but when I catch it then most of the debug features of the better_errors gem don't work. So it's either ugly unreadable error message but full better_errors features when you go afterwards to /__better_errors or nicely formatted backtrace and error message but without all the other features that /__better_errors offers like open the editor at the place where the error occurred.